nusskylab/nusskylab
# Implemented UI to add Admin for a Year

## Status
IN DEVELOPMENT 

## Migrations
NO

## Description
UI implemented to add an user by user-id as an Admin in a future year. No longer have to do this insertion via psql. However, still unable to add an user as Admin for a range of years in one go.

## Screenshots
#### Before:
![image](https://user-images.githubusercontent.com/16080561/75510963-90ee9280-5a27-11ea-8feb-f4074f12626a.png)


#### After:
![image](https://user-images.githubusercontent.com/16080561/75510972-9946cd80-5a27-11ea-84e8-0ddddb0b5265.png)


## Related PRs
List related PRs against other branches:
#503 

## Todos


## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Steps to Test or Reproduce
1. Select an user id from the drop down to add as Admin
2. Enter a year from 2020-2100 for which that user will be an Admin
3. In the psql database, check the admin table. An entry with the user id and the year should be added.
